gapic/templates/%namespace/%name_%version/%sub/services/%service/_client_macros.j2

```diff
@@ -141,7 +146,11 @@
 
         # Wrap the RPC method; this adds retry and timeout information,
         # and friendly error handling.
-        rpc = self._transport._wrapped_methods[self._transport.{{ method.transport_safe_name|snake_case}}]
+        if raw_response_callback:
+            call_type = "with_call"
+        else:
+            call_type = "default"
+        rpc = self._transport._wrapped_methods[self._transport.{{ method.transport_safe_name|snake_case}}][call_type]
```

Comment:
FWIW, I'm still against putting a dict in `_wrapped_methods`. I think we could just make it always do `with_call`, and optionally drop the `call` argument. Let me know if I'm missing why that would cause issues

If we have to have two _wrapped_methods, I'd suggest storing it as a separate `wrapped_methods_with_call` variable, to avoid complexities around the nested dictionary lookup

Reply:
I'll include another prototype where we have `with_call` unconditionally. I'll update this prototype to use a separate `wrapped_methods_with_call` variable

Reply:
I updated this prototype to use `wrapped_methods_with_call`. I'l investigate using `with_call` unconditionally in a separate prototype